Mathlib/GroupTheory/Coset/Defs.lean

```diff
@@ -124,12 +124,10 @@ def quotientRightRelEquivQuotientLeftRel : Quotient (QuotientGroup.rightRel s) 
     Quotient.map' (fun g => g⁻¹) fun a b => by
       rw [leftRel_apply, rightRel_apply]
       exact fun h => (congr_arg (· ∈ s) (by simp)).mp (s.inv_mem h)
-      -- Porting note: replace with `by group`
```

Comment:
Are these WONTFIX?

Reply:
I think `by group` is not a big improvement over `by simp` that we need to keep around a porting note for.

Reply:
So yes, I propose WONTFIX.